src/DocumentFormat.OpenXml/GeneratedCode/schemas-microsoft-com_office_excel.g.cs

```diff
@@ -199,6 +200,13 @@ public ClientData(string outerXml) : base(outerXml)
         [Index(0)]
         public EnumValue<DocumentFormat.OpenXml.Vml.Spreadsheet.ObjectValues> ObjectType { get; set; }
 
+        private static readonly ISemanticConstraint[] _semanticConstraint = new ISemanticConstraint[] {
```

Comment:
In [corefx/coding-style.md at master · dotnet/corefx](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md ) NO.1, we use Allman style braces, where each brace begins on a new line. A single line statement block can go without braces but the block must be properly indented on its own line and must not be nested in other statement blocks that use braces (See issue 381 for examples). One exception is that a using statement is permitted to be nested within another using statement by starting on the following line at the same indentation level, even if the nested using contains a controlled block.

Reply:
Yup, and I spent about 20 minutes trying to get the generator to produce that to no avail. Decided in this case it's fine; I have future plans to streamline the declaration of this metadata even more, so it'll change again in the next few months.